README.rst

```diff
@@ -109,13 +109,26 @@ Global options
     `--source-pipeline=videotestsrc`. A script like `press("18")` will change
     videotestsrc's pattern property (see `gst-inspect videotestsrc`).
 
-  x11:<display>
+  x11:<display>,<key_mapping_file>
```

Comment:
Yes, display will typically be of the form `:0`, so that's why a comma is used.  display is also optional so I'll update that too.